app/src/main/java/com/lec/spring/controller/RecipeController.java

```diff
@@ -0,0 +1,79 @@
+package com.lec.spring.controller;
+
+import com.lec.spring.entity.Recipe;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/recipe")
+public class RecipeController {
+
+
+    private static final Map<Long, Recipe> recipes = new HashMap<>();
+    private static Long id = 0L;
```

Comment:
이 로직을 Thread-safe하게 만들려면 어떻게 해야할까요?

```suggestion
    private final ConcurrentHashMap<Long, Recipe> recipes = ...;
    private AtomicLong id = ...;
```

이 부분은 공부해서 알려주세요.